specs/CruiseErrorMessages.spec

```diff
@@ -0,0 +1,54 @@
+
+CruiseErrorMessages
```

Comment:
Original spec has steps to validate errors and warnings on the stage details page too - https://github.com/gocd/functional-tests/blob/master/manual/CruiseErrorMessages.spec#L110 This spec should also have that validation